shell/platform/android/platform_view_android.h

```diff
@@ -53,6 +53,16 @@ class PlatformViewAndroid final : public PlatformView {
                       std::shared_ptr<PlatformViewAndroidJNI> jni_facade,
                       bool use_software_rendering);
 
+  //----------------------------------------------------------------------------
+  /// @brief      Creates a new PlatformViewAndroid but using an existing
+  ///             Android GPU context to create new surfaces. This maximizes
+  ///             resource sharing between 2 PlatformViewAndroids of 2 Shells.
+  ///
+  PlatformViewAndroid(PlatformView::Delegate& delegate,
+                      flutter::TaskRunners task_runners,
+                      std::shared_ptr<PlatformViewAndroidJNI> jni_facade,
```

Comment:
nit: Should pass these by const reference.  The copy constructor for std::shared_ptr is pretty heavy, it includes mutex locking and stuff.

Reply:
done here and in android_shell_holder